src/server/index.tsx

```diff
@@ -46,6 +49,54 @@ server.get("/robots.txt", async (_req, res) => {
   res.send(robotstxt);
 });
 
+server.get("/css/themes-list", async (req, res) => {
+  const builtinThemes = [
+    "litera",
+    "materia",
+    "minty",
+    "solar",
+    "united",
+    "cyborg",
+    "darkly",
+    "journal",
+    "sketchy",
+    "vaporwave",
+    "vaporwave-dark",
+    "i386",
+    "litely",
+    "nord",
+  ];
```

Comment:
You should have access to the shared exported themes already: https://github.com/LemmyNet/lemmy-ui/blob/main/src/shared/utils.ts#L78